test/spark/app/src/main/scala/Sonnets.scala

```diff
@@ -9,10 +9,17 @@ object Sonnets {
     spark.sparkContext.hadoopConfiguration.set("fs.s3a.endpoint", "s3.docker.lakefs.io:8000")
     spark.sparkContext.hadoopConfiguration.set("fs.s3a.connection.ssl.enabled", "false")
     var sc = spark.sparkContext
-    sc.setLogLevel("DEBUG")
-    val sonnets = sc.textFile("s3a://example/master/sonnets.txt")
-    val counts = sonnets.flatMap(line => line.split(" ")).map(word => (word, 1)).reduceByKey(_ + _)
-    counts.saveAsTextFile("s3a://example/master/sonnets-wordcount")
-    spark.stop()
+    sc.setLogLevel("INFO")
+    import spark.implicits._
+    val sonnets = sc.textFile("s3a://example/main/sonnets.txt")
+    val counts = sonnets.flatMap(line => line.split(" ")).map(word => (word, 1)).reduceByKey(_ + _).toDF
+    counts.write.format("csv").save("s3a://example/main/sonnets-wordcount.csv")
+
+    val wc = spark.read.format("csv").load("s3a://example/main/sonnets-wordcount.csv")
+    val life = wc.filter(x => x(1) == "42").map(x => x(0).toString).collect.sorted.mkString(",")
+    if (life != "can,or") {
```

Comment:
Is there a chance for "or,can" randomness?

Reply:
sorted the results ;)